src/sentry/common.ts

```diff
@@ -18,6 +25,35 @@ export function getCommonClientOptions(dsn: string, themeName: string): Paramete
             // This error is caused by `beacon.min.js` polyfill in Cloudflare Insights. From what we see, it doesn't affect the user experience.
             'Illegal invocation',
         ],
+
+        // Ignore global code errors that are not related to the application code.
+        // See comments in `IGNORED_EVENT_CULPRITS` const for context on each entry
+        beforeSend: (event) => {
```

Comment:
Do you think we could use the [`allowUrls`](https://github.com/prezly/prezly/blob/ac85b8621f5bbfc728972fcd1ea12f5e30968a1c/apps/press/modules/widget/templates/_raven.php#L16-L18) to whitelist our domain only? That way, erros from scripts from other URLs should be ignored.

Reply:
It would help us, if we had a pre-determined domain, but since themes are multi-tenancy applications, we can't know the domain before we receive the request. And this configuration is executed when the application starts, so we can't get the domain at that stage.

Reply:
I think you can match partially, so you don't need a full url. Maybe we could have something like `/_next/static/chunks/*`? 🤔 

> As a result, if you add 'foo.com' to it, it will also match on https://bar.com/myfile/foo.com.

See the [documentation](https://docs.sentry.io/platforms/javascript/configuration/options/#allow-urls).

Reply:
Still not sure, but prolly worth investigating later. My concern is that it won't filter these particular issues, since technically they're happening right on the page, so they would probably pass the URL check.